[doc] Add known issue to all affected versions due to jackson parsing bug (#93915)
elastic/elasticsearch: docs/reference/release-notes/8.0.0.asciidoc

bin/elasticsearch-create-enrollment-token -s kibana
----
--
// tag::jackson-filtering-bug[]

> * When parsing request and when the last element in an array is filtered out
> (for instance using _source.include option) the parsing will fail. This
> is due to a bug in Jackson parser https://github.com/FasterXML/jackson-core/issues/882
> Fixed in {es} 8.6.1 ({es-pull}92480[#91456])

Reviewer: ```suggestion
* Parsing a request when the last element in an array is filtered out
(for instance using `_source_includes`) fails. This
is due to https://github.com/FasterXML/jackson-core/issues/882[a bug in Jackson parser].
Fixed in {es} 8.6.1 ({es-pull}92480[#91456])
```
